samples/stock/demo/relative-rotation-graph/demo.html

```diff
@@ -0,0 +1,7 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/modules/series-label.js"></script>
+<script src="https://code.highcharts.com/modules/annotations.js"></script>
+<script src="https://code.highcharts.com/stock/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container"></div>
```

Comment:
As a non-finance person, it's not clear to me what the chart shows. Can you add a short description of the chart and what the data represents? With a link to the algorithm. See existing demos for template:

https://github.com/highcharts/highcharts/blob/794655bde5a497579a952b30dc77651ca5e12bb1/samples/highcharts/demo/area-basic/demo.html#L8-L12